src/core/FxMixer.cpp

```diff
@@ -336,6 +336,11 @@ void FxMixer::deleteChannel( int index )
 		deleteChannelSend( ch->m_receives.first() );
 	}
 
+	// if m_lastSoloed was our index, reset it
+	if (m_lastSoloed == index) { m_lastSoloed = -1; }
+	// if m_lastSoloed is > delete index, it will move left
+	else if (m_lastSoloed > index) { qMax(0, --m_lastSoloed); }
```

Comment:
I think you don't need the `qMax()`.

Reply:
> 
> 
> I think you don't need the `qMax()`.

I believed so, too, after I committed. I wanted to get a second opinion though. 😄